uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceTab.tpl

```diff
@@ -1,10 +1,52 @@
 <section data-leave-absences-absence-tab>
-  <ui-view></ui-view>
+  <div class="absence-tab-page">
```

Comment:
Please mark the piece of code which needs to be moved in the next ticket with comments.